Source/Core/UICommon/GameFileCache.h

```diff
@@ -26,8 +24,12 @@ std::vector<std::string> FindAllGamePaths(const std::vector<std::string>& direct
 class GameFileCache
 {
 public:
+  GameFileCache();  // Uses the default path
+  GameFileCache(std::string path);
+
   void ForEach(std::function<void(const std::shared_ptr<const GameFile>&)> f) const;
 
+  size_t Size() const;
```

Comment:
Don't we usually use ``GetSize()`` or something? Also it might be better to name it count or something because size is a bit ambiguous.

Reply:
You're right, `GetSize` is what we normally use.